client-lite/build/postinst.in.sh

```diff
@@ -43,11 +43,12 @@ configure_dir() {
     fi
     chgrp -R $do_group_name $dir_path
     chown $do_user_name $dir_path
-    chmod g+w $dir_path
 }
 
 configure_dir "$config_path"
-configure_dir "$persistence_path"
+# group needs write permission to config path (SDK writing configs)
+chmod g+w $config_path
```

Comment:
possible to still have this within configure_dir()? where configure_dir() accepts a bool param to check whether it should toggle group write

Reply:
Not worth it I felt.